deprecated-react-native-prop-types/package.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "deprecated-react-native-prop-types",
-  "version": "3.0.1",
+  "version": "4.0.0",
```

Comment:
I think this can be v3.0.2. What user perceivable changes are there from renaming this package?

Reply:
I discussed this with @cortinico, but adding this comment for posterity.

We want to bump this to `4.0.0` because `react-native@0.71` will still use `@react-native/normalize-color` and we want users of `0.71` to continue depending on `3.0.1`. (Although there is probably no observable difference for 99% of users, it *could* cause two instances of `@react-native/normalize-color` to be instantiated in an app.)

So `3.x.x` is effectively the `react-native@0.71` line, and `4.x.x` is the `react-native@0.72+` line.

We should incorporate this in the `CHANGELOG.md`.

Reply:
Correct. That's because we use the caret here: https://github.com/facebook/react-native/blob/3f2691cf84b2f7852c55a58b45194329728e1c70/package.json#L121

Reply:
I've included it in the changelog